src/webots/nodes/WbMesh.cpp

```diff
@@ -0,0 +1,415 @@
+// Copyright 1996-2020 Cyberbotics Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "WbMesh.hpp"
+
+#include "WbAffinePlane.hpp"
+#include "WbBoundingSphere.hpp"
+#include "WbField.hpp"
+#include "WbFieldChecker.hpp"
+#include "WbMFString.hpp"
+#include "WbNodeUtilities.hpp"
+#include "WbRay.hpp"
+#include "WbResizeManipulator.hpp"
+#include "WbSimulationState.hpp"
+#include "WbTransform.hpp"
+#include "WbTriangleMesh.hpp"
+#include "WbUrl.hpp"
+#include "WbVector2.hpp"
```

Comment:
Could you please check again these includes?
Some of them seems useless (for example WbNodeUtilities, but maybe others too)

Reply:
Good point, I reviewed the headers and severa of them can be removed.